### PR TITLE
Repaired and extended support for help page translations including ta…

### PR DESCRIPTION
**Pull request by anaqreon**
…ble of contents files at the top level. Moved help module javascript to mod_help.js.

### CODE REVIEW

File: include/help.php [hunk 1]
```diff
@@ -107,20 +107,39 @@ function preg_callback_help_include($matches) {
...
+	$lang = $x['language'];
+	$url_idx = ($x['from_url'] ? 1 : 0);
+	if($x['from_url'] && $lang !== 'en') {
+		$path .= '/' . $lang;	
+	}
+		
 	$b = basename($s);
-	$d = dirname($s);
-
-	if($dirname !== '-') {
```
Comment (zotlabs) [inline include/help.php]: I didn't see this in the replacement code. It's needed for rare cases when for instance your chosen language is (for instance) 'de' but you're editing and viewing files in english. Without it there is no way to load the english version.

Comment (anaqreon) [inline include/help.php]: The `$dirname` was not actually defined in that function. I assume it was supposed to be `$d`. In any case, the new code loads the language specified in the URL if present (e.g. `/help/de/member/member_guide`), falling back to the browser-preferred language and then English.

Comment (zotlabs) [inline include/help.php]: Understood - there was a bug;  however my original premise still stands. Can you load the English version specifically if your browser language is something other than English? Without the '-' hack you can't.

Comment (anaqreon) [inline include/help.php]: You can definitely load the English version (or any other translation) by explicitly putting the language in the URL, which overrides the browser preference.

I did find a mistake in my code, though, so let me fix it and update the pull request before we merge it.

**Pull request merged**